lib/widgets/infinite_scroll.dart

```diff
@@ -54,9 +58,15 @@ class InfiniteScroll<T> extends HookWidget {
     final page = data.value.length ~/ batchSize + 1;
 
     return ListView.builder(
-      // +1 for the loading widget
-      itemCount: data.value.length + 1,
+      padding: padding,
+      // +2 for the loading widget and prepend widget
+      itemCount: data.value.length + 2,
       itemBuilder: (_, i) {
+        if (i == 0) {
+          return prepend ?? Container();
```

Comment:
Set default instead of null coalescing

Reply:
![image](https://user-images.githubusercontent.com/10037914/94603301-5804ed80-0296-11eb-9f3c-5f90e6522321.png)
can't do unfortunately :(

Reply:
const Container()